forbidigo/forbidigo.go

```diff
@@ -190,6 +216,47 @@ func (v *visitor) textFor(node ast.Node) string {
 	return buf.String()
 }
 
+// pkgTextFor expands the selector in a selector expression to the full package
+// name and (for variables) the type:
+//
+// - example.com/some/pkg.Function
+// - example.com/some/pkg.CustomType.Method
+//
+// It returns an empty string for other expressions.
+func (v *visitor) pkgTextFor(node ast.Node) string {
```

Comment:
I admittedly have a minor preference for not using the empty string to communicate that nothing was found but I don't feel particularly strongly about it.  Other options could include an extra bool return value or returning a pointer.

Reply:
Let's return a pointer - changed.